programs/svm-spoke/src/instructions/admin.rs

```diff
@@ -241,10 +244,15 @@ pub fn relay_root_bundle(
     let root_bundle = &mut ctx.accounts.root_bundle;
     root_bundle.relayer_refund_root = relayer_refund_root;
     root_bundle.slow_relay_root = slow_relay_root;
+
+    emit_cpi!(RelayRootBundleEvent {
+        relayer_refund_root,
+        slow_relay_root,
+        root_bundle_id: state.root_bundle_id,
+    });
+
+    // Finally, increment the root bundle id
```

Comment:
not directly related to this PR, but I think its confusing to have prop named `root_bundle_id` in state account. Instead it should be something to resemble number of relayed bundles or `next_root_bundle_id` - Maybe add this as TODO here